backend/src/main/java/shook/shook/member/application/MemberService.java

```diff
@@ -29,4 +29,8 @@ public Member register(final String email) {
     public Optional<Member> findByEmail(final Email email) {
         return memberRepository.findByEmail(email);
     }
+
+    public Optional<Member> findById(final Long id) {
+        return memberRepository.findById(id);
```

Comment:
```suggestion
        return memberRepository.findById(id)
.orElseThrow(MemberNotExsistException::new);
```

예외 처리는 서비스가 해주는 게 맞는 것 같아요!

Reply:
넵 알겠습니다! 저도 memberService에서 예외를 처리하면 좋을 것 같습니다:)